indica/models/helike_spectroscopy.py

```diff
@@ -475,29 +481,30 @@ def plot(self):
         plt.figure()
         channels = self.los_transform.x1
         cols_time = cm.gnuplot2(np.linspace(0.1, 0.75, len(self.t), dtype=float))
-        if "spectra" in self.bckc.keys():
-            spectra = self.bckc["spectra"]
-            if "channel" in spectra.dims:
-                spectra = spectra.sel(channel=np.median(channels))
+        if "intens" in self.bckc.keys():
+            intens = self.bckc["intens"]
+            if "channel" in intens.dims:
+                intens = intens.sel(channel=np.median(channels))
             for i, t in enumerate(np.array(self.t, ndmin=1)):
                 plt.plot(
-                    spectra.wavelength,
-                    spectra.sel(t=t),
+                    intens.wavelength,
+                    intens.sel(t=t),
                     color=cols_time[i],
                     label=f"t={t:1.2f} s",
                 )
-            plt.ylabel("Brightness (a.u.)")
+            plt.ylabel("Intensity (count/s.)")
```

Comment:
For the future, this should be dealt with automatically by DataArray now that longname and units are included.